mlir/lib/Dialect/MIOpen/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -2331,9 +2336,8 @@ struct GridwiseGemmV2RewritePattern
     int64_t numBlksPerXdlops = (MPerXdlops * NPerXdlops) / (m * n);
     const auto &tailResults = blockwiseGemmV2TailOp->getResults();
     int64_t wavesInKernelBlock = kernelBlockSize / waveSize;
-    int64_t resultCVectorLen = vectorType.getNumElements();
-    int64_t numElements = resultCVectorLen * tailResults.size();
 
+    int64_t numElements = regCVectorLen * vectorNumber;
```

Comment:
Can we name `vectorNumber` something like `numVectors`?

Reply:
Yes agreed that this is an awkward name, but I was following the naming convention from `XdlopsSelection.h`. To limit the scale of refactoring in this PR, I'd rename the local variable name and leave the more comprehensive renaming to when we touch `XdlopsSelection.h`

Reply:
Agreed